tests/page-objects/privacy-policy/privacy-policy.wdio.page.js

```diff
@@ -0,0 +1,39 @@
+const utils = require('../../utils');
+const privacyWrapper = () => $('#privacy-policy-wrapper');
+const privacyAccept = async () => (await privacyWrapper()).$('.btn');
+const privacyConfig = () => $('.privacy-policy.configuration');
+
+const goToPrivacyPolicyConfig = async () => {
+  await browser.url(utils.getBaseUrl() + 'privacy-policy');
+  const privacyPolicyContainer = await privacyConfig();
+  await privacyPolicyContainer.waitForDisplayed();
+  return privacyPolicyContainer.getText();
+};
+
+const getPrivacyPolicyFromOverlay = async () => {
+  (await privacyWrapper()).waitForDisplayed();
+  return (await privacyWrapper()).getText();
+};
+
+const acceptPrivacyPolicy = async () => {
+  return (await privacyAccept()).click();
+};
+
+const updatePrivacyPolicy = async (docId, policyKey, policyText) => {
```

Comment:
Can we make the `fr` not hardcoded here and actually pass it as a param? Also `policyKey` is the attachment name, right?